editor.planx.uk/src/pages/FlowEditor/lib/analyticsProvider.tsx

```diff
@@ -171,6 +176,30 @@ export const AnalyticsProvider: React.FC<{ children: React.ReactNode }> = ({
     }
   }
 
+  async function trackNextStepsLinkClick(metadata?: NextStepsLinkMetadata) {
+    if (shouldTrackAnalytics && lastAnalyticsLogId) {
+      await publicClient.mutate({
+        mutation: gql`
+          mutation UpdateHasClickNextStepsLink(
```

Comment:
Can I just define mutations like this? In my previous experience this would have to match a mutation via Graphene although is there Hasura magic going on? It seems to work which honestly surpised me 😅

Reply:
Yep! This is allowed and should work just fine. 

We don't currently enforce an allow-list of queries/mutations though it is something Hasura supports (and recommends in production). Docs: https://hasura.io/docs/latest/security/allow-list/